app/src/main/java/ch/epfl/sweng/rps/db/FirebaseHelper.kt

```diff
@@ -18,5 +19,87 @@ sealed class FirebaseHelper {
                 uid = uid,
             )
         }
+
+        suspend fun getStatsData (selectMode: Int): MutableList<List<String>> {
+            val userGameList = FirebaseRepository().gamesOfUser(FirebaseRepository().getCurrentUid())
+            val statsResult: MutableList<String> = ArrayList()
+            val allStatsResult: MutableList<List<String>> = java.util.ArrayList()
```

Comment:
You can create an ArrayList with `mutableListOf<String>()`

Reply:
changed! thanks